com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -1610,6 +1610,16 @@ void PrepareGPULightdata(CommandBuffer cmd, HDCamera hdCamera, CullingResults cu
             }
         }
 
+        void ClearUnusedProcessedReferences(CullingResults cullResults, HDProbeCullingResults hdProbeCullingResults)
+        {
+            for (int i = cullResults.visibleLights.Length; i < m_ProcessedLightData.size; i++)
```

Comment:
severity: Error
	message: C:\build\output\Unity-Technologies\Graphics\com.unity.render-pipelines.high-definition\Runtime\Lighting\LightLoop\LightLoop.cs(1615,64): error CS0103: The name 'm_ProcessedLightData' does not exist in the current conte
	
	With @kecho change this don't exist anymore. @kecho  can you look at this PR know what we need to change?